alicloud/cpi.yml

```diff
@@ -0,0 +1,107 @@
+---
+# CPI
+- type: replace
+  path: /releases/-
+  value:
+    name: bosh-alicloud-cpi
+    version: 16
+    url: http://bosh-((region)).oss-((region)).aliyuncs.com/bosh-alicloud-cpi-release-r16.tgz
+    sha1: dc63a7b28fa77294ae4c7305c85d9590ca690cd9
+
+# light stemcell
+- type: replace
+  path: /resource_pools/name=vms/stemcell?
+  value:
+    url: http://bosh-((region)).oss-((region)).aliyuncs.com/light-bosh-stemcell-1016-alicloud-kvm-ubuntu-trusty-go_agent.tgz
+    sha1: ef7dfcf7d677a32625631cdc49257b8680eca9e6
+
+# Configure ECS sizes
+- type: replace
+  path: /resource_pools/name=vms/cloud_properties?
+  value:
+    availability_zone: ((zone))
+    instance_type: "ecs.mn4.small"
+    instance_name: "bosh-director"
+    ephemeral_disk: {size: "51_200", category: "cloud_efficiency"}
+    system_disk: {size: "51_200", category: "cloud_efficiency"}
+    key_pair_name: ((key_pair_name))
+
+- type: replace
+  path: /disk_pools/name=disks/cloud_properties?
+  value:
+    type: "cloud_efficiency"
+    instance_role: "director"
+
+- type: replace
+  path: /networks/name=default/subnets/0/cloud_properties?
+  value:
+    vswitch_id: ((vswitch_id))
+    security_group_ids: [((security_group_id))]
+    internet_charge_type: "PayByTraffic"
```

Comment:
can `internet_charge_type` also be defaulted in the cpi?

Reply:
CPI does not set default for `internet_charge_type `, but Alicloud API set a default 'PayByTraffic'.